*** ./smali/com/android/internal/widget/ILockSettings.smali	2016-06-27 21:53:50.471285939 +0800
--- /home/kingrealer/MIUI/N958St/temp/new_smali/framework2.jar.out/./smali/com/android/internal/widget/ILockSettings.smali	2016-06-27 21:56:57.137491186 +0800
***************
*** 111,116 ****
--- 111,124 ----
      .end annotation
  .end method
  
+ .method public abstract setRawLockPassword([BI)V
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
  .method public abstract setString(Ljava/lang/String;Ljava/lang/String;I)V
      .annotation system Ldalvik/annotation/Throws;
          value = {
