*** ./smali/android/app/ActivityOptions.smali	2016-06-27 21:54:38.831850094 +0800
--- /home/kingrealer/MIUI/N958St/temp/new_smali/framework.jar.out/./smali/android/app/ActivityOptions.smali	2016-06-27 21:58:15.230433070 +0800
***************
*** 961,963 ****
--- 961,979 ----
          :pswitch_2
      .end packed-switch
  .end method
+ 
+ .method public static makeClipRevealAnimation(Landroid/view/View;IIII)Landroid/app/ActivityOptions;
+     .locals 1
+     .param p0, "source"    # Landroid/view/View;
+     .param p1, "startX"    # I
+     .param p2, "startY"    # I
+     .param p3, "width"    # I
+     .param p4, "height"    # I
+ 
+     .prologue
+     invoke-static {p0, p1, p2, p3, p4}, Landroid/app/ActivityOptions;->makeScaleUpAnimation(Landroid/view/View;IIII)Landroid/app/ActivityOptions;
+ 
+     move-result-object v0
+ 
+     return-object v0
+ .end method
