*** ./smali/com/android/server/AppWidgetServiceImpl.smali	2016-06-27 21:55:03.556140599 +0800
--- /home/kingrealer/MIUI/N958St/temp/new_smali/services.jar.out/./smali/com/android/server/AppWidgetServiceImpl.smali	2016-06-27 21:58:39.946733128 +0800
***************
*** 4420,4425 ****
--- 4420,4433 ----
      .end local v2    # "stream":Ljava/io/FileInputStream;
      :cond_0
      :goto_0
+     iget-object v3, p0, Lcom/android/server/AppWidgetServiceImpl;->mContext:Landroid/content/Context;
+ 
+     iget-object v4, p0, Lcom/android/server/AppWidgetServiceImpl;->mInstalledProviders:Ljava/util/ArrayList;
+ 
+     iget v5, p0, Lcom/android/server/AppWidgetServiceImpl;->mUserId:I
+ 
+     invoke-static {v3, v4, v5}, Lcom/android/server/AppWidgetServiceImplInjector;->updateWidgetPackagesLocked(Landroid/content/Context;Ljava/util/List;I)V
+ 
      return-void
  
      .restart local v2    # "stream":Ljava/io/FileInputStream;
***************
*** 7293,7299 ****
--- 7301,7317 ----
  
      if-nez v3, :cond_0
  
+     return-void
+ 
      :goto_0
+     iget-object v3, p0, Lcom/android/server/AppWidgetServiceImpl;->mContext:Landroid/content/Context;
+ 
+     iget-object v4, p0, Lcom/android/server/AppWidgetServiceImpl;->mInstalledProviders:Ljava/util/ArrayList;
+ 
+     iget v5, p0, Lcom/android/server/AppWidgetServiceImpl;->mUserId:I
+ 
+     invoke-static {v3, v4, v5}, Lcom/android/server/AppWidgetServiceImplInjector;->updateWidgetPackagesLocked(Landroid/content/Context;Ljava/util/List;I)V
+ 
      return-void
  
      :cond_0
