*** ./smali/android/net/wifi/SupplicantStateTracker$CompletedState.smali	2016-06-27 21:54:04.127444680 +0800
--- /home/kingrealer/MIUI/N958St/temp/new_smali/framework.jar.out/./smali/android/net/wifi/SupplicantStateTracker$CompletedState.smali	2016-06-27 21:57:18.473747541 +0800
***************
*** 71,76 ****
--- 71,78 ----
      # setter for: Landroid/net/wifi/SupplicantStateTracker;->mNetworksDisabledDuringConnect:Z
      invoke-static {v0, v1}, Landroid/net/wifi/SupplicantStateTracker;->access$602(Landroid/net/wifi/SupplicantStateTracker;Z)Z
  
+     invoke-static {}, Landroid/net/wifi/SupplicantStateTrackerInjector;->handleNetworkConnectionComplete()V
+ 
      :cond_0
      return-void
  .end method
