*** ./smali/android/net/wifi/WifiConfiguration$PairwiseCipher.smali	2016-06-27 21:54:06.643473976 +0800
--- /home/kingrealer/MIUI/N958St/temp/new_smali/framework.jar.out/./smali/android/net/wifi/WifiConfiguration$PairwiseCipher.smali	2016-06-27 21:57:21.429783118 +0800
***************
*** 31,37 ****
      .locals 3
  
      .prologue
!     const/4 v0, 0x3
  
      new-array v0, v0, [Ljava/lang/String;
  
--- 31,37 ----
      .locals 3
  
      .prologue
!     const/4 v0, 0x4
  
      new-array v0, v0, [Ljava/lang/String;
  
***************
*** 53,58 ****
--- 53,64 ----
  
      aput-object v2, v0, v1
  
+     const/4 v1, 0x3
+ 
+     const-string v2, "SMS4"
+ 
+     aput-object v2, v0, v1
+ 
      sput-object v0, Landroid/net/wifi/WifiConfiguration$PairwiseCipher;->strings:[Ljava/lang/String;
  
      return-void
